dotnet-desktop-guide/framework/wpf/controls/listview-overview.md

```diff
@@ -12,7 +12,7 @@ ms.assetid: 989e12b0-260e-4570-95c6-489284003ce2
 ---
 # ListView Overview
 
-The <xref:System.Windows.Controls.ListView> control provides the infrastructure to display a set of data items in different layouts or views. For example, a user may want to display data items in a table and also to sort its columns.  
+The <xref:System.Windows.Controls.ListView> control provides the infrastructure to display a set of data items in multiple layouts or views. For example, a user may want to display data items in a table and also to sort its columns.  
```

Comment:
You are probably correct that _different_ is appropriate. I am not familiar with the topic enough to be sure. Many people use the word _different_ improperly. It might be being used properly here. Due to the misuse, the word seems ambiguous to me here. If there is a way to differentiate what is meant then that probably would help. Perhaps it could be simplified to _to display a set of data items in other ways_ or _to display a set of data items isoug a different layout or view_ .

Reply:
I like that suggestion. I'll implement a change here. Thanks for the feedback!